diff --git a/src/ackermann_mux.cpp b/src/ackermann_mux.cpp
index 79d8d53..f53f405 100755
--- a/src/ackermann_mux.cpp
+++ b/src/ackermann_mux.cpp
@@ -88,7 +88,7 @@ void AckermannMux::init()
   /// Publisher for output topic:
   cmd_pub_ =
     this->create_publisher<ackermann_msgs::msg::AckermannDriveStamped>(
-    "ackermann_cmd",
+    "ackermann_drive_out",
     rclcpp::QoS(rclcpp::KeepLast(1)));
 
   /// Diagnostics:
